src/chronometer.js

```diff
@@ -1,33 +1,69 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+    if(typeof printTimeCallback === 'function'){ 
```

Comment:
Since they say in the instructions that you are receiving a callback, there's an easier way to do this.
```
start(printTimeCallback) {
    // ... your code goes here
    this.intervalId = setInterval(() => {
      this.currentTime++;
      if (printTimeCallback) {
        printTimeCallback();
      }
    }, 1000);
  }
```